src/wallet/api/wallet.cpp

```diff
@@ -1370,6 +1376,15 @@ size_t WalletImpl::importMultisigImages(const vector<string>& images) {
     return 0;
 }
 
+size_t WalletImpl::importMultisigImages(string filename) {
```

Comment:
Why copy ?

Reply:
the first `importMultisigImages` function works with hex strings, this one works with raw data blobs

Reply:
I do not understand the relevance.